apps/cli/src/commands/access-control/add-admin.ts

```diff
@@ -1,40 +1,55 @@
-import Vorpal from "vorpal";
-import { addAddressToRole, getSignerFromPrivateKey } from "@sentry/core";
+import { Command } from 'commander';
+import inquirer from 'inquirer';
+import { addAddressToRole, getSignerFromPrivateKey } from '@sentry/core';
 
 /**
  * Function to add an admin to the Referee contract.
- * @param cli - Vorpal instance
+ * @param program - Commander instance
  */
-export function addAdmin(cli: Vorpal) {
-    cli
-        .command('add-admin', 'Adds an address to the DEFAULT_ADMIN_ROLE in the Referee contract.')
-        .action(async function (this: Vorpal.CommandInstance) {
-            const {address} = await this.prompt({
-                type: 'input',
-                name: 'address',
-                message: 'Address to be added to the DEFAULT_ADMIN_ROLE:' 
-            });
-
-            const {privateKey} = await this.prompt({
-                type: 'password',
-                name: 'privateKey',
-                message: 'Private key of the current admin address. Must have DEFAULT_ADMIN_ROLE. Check by calling get-list-of-admins:',
-                mask: '*',
-            });
-
-            if (!address || !privateKey) {
-                this.log('Both address and private key are required.');
-                return;
-            }
+export function addAdmin(program: Command) {
+    program
```

Comment:
This could still be named cli and we don't have to change that in every file ?

Reply:
Yes, this has been done.